03-task-manager/starter/db/connect.js

```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+
+const connectDB = (url) => {
+  mongoose.set('strictQuery', false);
+  return mongoose.connect(url);
```

Comment:
Just curious why you set this particular value? 

In the video I believe he uses the following options:

```
const connectDB = (url) => {
  return mongoose.connect(url, {
    useNewUrlParser: true,
    useCreateIndex: true,
    useFindAndModify: false,
    useUnifiedTopology: true,
  });
};
```

  
Those are explained further here in the docs for mongoose V5: https://mongoosejs.com/docs/5.x/docs/connections.html#options